Package.swift

```diff
@@ -0,0 +1,48 @@
+// swift-tools-version:5.3
```

Comment:
```suggestion
// swift-tools-version:5.7
```

The convention is to target the current stable Xcode version's Swift, which is 5.7.